src/components/Albums/AlbumList/AlbumList.js

```diff
@@ -0,0 +1,96 @@
+import { AlbumCard } from "../AlbumCard/AlbumCard";
+import albumListTemplate from "./AlbumList.html";
+
+export class AlbumList {
+  constructor(mountPoint, props = []) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  querySelectors() {
+    this.albumsList = this.mountPoint.querySelector(".album-list__container");
+  }
+
+  mountChildren() {
+    const albums = [
+      {
+        albumName: "Loud",
+        artistName: "by Rihanna",
+        sourceImage: 'src="./assets/img/AlbumsCovers/rihanna_loud.jpg"'
```

Comment:
really? 'src="./assets/img/AlbumsCovers/rihanna_loud.jpg"' ?? You should not have this `src=` prefix in there

Reply:
Corrected